### PR TITLE
Converter for 9K to 9000

### PR DESCRIPTION
**Pull request by MrZeffy**
Hey there, i saw your issue listed for this project. I created a function that you can call for converting it. I've never worked with TypeScript and i couldn't figure out how to test this solution. But the function is working fine. You can place this function in a separate file maybe, and call it wherever you need it. I implemented in the parser.ts

Hope this helps.

Also, this works only with capitalized letters(K, M, B). If you get small alphabets in response from scraper, you can convert the string to uppercase or just replace the switch statement with if-else.

Happy coding :)

### CODE REVIEW

File: src/parser.ts [hunk 1]
```diff
@@ -103,12 +103,64 @@ const getVideoCount = (channel: any) => {
  */
 const getSubscriberCount = (channel: any) => {
     try {
-        return channel.subscriberCountText.simpleText.split(' ').shift();
+        let count = channel.subscriberCountText.simpleText.split(' ').shift();
+        let lastChar = count.slice(-1);
+        if (parseInt(lastChar) === NaN) {
```
Comment (DrKain) [inline src/parser.ts:108]: This statement will almost never be true, when checking if something is NaN you may want to use `isNaN()`  
This error means subscriber counts like `200` or `16` were cut down to `20` and `1`
  
https://stackoverflow.com/questions/825402/why-does-isnan-string-with-spaces-equal-false

Comment (DrKain) [reply src/parser.ts:108]: I can fix this in my branch but I figured I'd let you know either way

**Comment by DrKain** [comment]
For the most part it looks good to me, I'll tweak it slightly, run some testing then push into the main branch. 
Thank you for the help

This merge will close #41

File: src/parser.ts (hunk 1 above)
Comment (MrZeffy) [reply src/parser.ts:108]: My bad. Never been into this error. Don't know how that function worked fine when i tested it though. 
Also, thanks for pointing that out😉.

**Pull request merged**